Calibration/IsolatedParticles/plugins/IsolatedTracksCone.cc

```diff
@@ -1080,7 +1358,7 @@ void IsolatedTracksCone::beginJob() {
   t_ievt = new std::vector<unsigned int>();
   t_ilum = new std::vector<unsigned int>();
```

Comment:
not part of this PR, but these vectors are never deleted

Reply:
These are part of a branch of a Tree. I am not sure if these to be deleted.